app/src/newdocs/content/streams.mdx

```diff
@@ -19,7 +33,61 @@ import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 <ScrollableAnchor id="work-with-streams-via-sdks"><div>
 
 ## Work with streams via SDKs
---content--
+
+The easiest way to work with streams is to use the JavaScript client, which works in `node.js` as well as the browser. See the <Link to={links.newdocs.SDKs}>SDKs section</Link> for the official and community-maintained client libraries available for various languages:
+
+- **JavaScript** client works in the browser as well as node.js.
+- **Java** client is work-in-progress but already implements the most important set of functionality.
```

Comment:
It should be obvious but might be worth mentioning the java client is server-side only?

Reply:
There is no mention of it in the lib repo so I left it out from here as well.

Reply:
:+1: